src/pass.js

```diff
@@ -284,6 +284,16 @@ class Pass {
 		return this;
 	}
 
+  logoText(text) {
+    this._props.logoText = text;
+    return this;
+  }
+
+  backgroundColor(colour) {
+    this._props.backgroundColor = colour;
+    return this;
+  }
+
```

Comment:
There is no need to add methods for this. Also, by doing this, you should get anyway the errors, because the changes must be done in schema.js.
Also, `backgroundColor` is already settable through overrides.
Also, again, you cannot add only these two props. If you add methods for overrides, you should add them for ALL. For this reason, I decided to not create single methods, due to increased complexity in manteinance.

Reply:
I was unable to set backgroundColor through overrides. I get an error about it not being of type string... Hence created a method... Maybe a small change to overrides would fix that though?

Reply:
That's weird. You should open a new issue for this so we can investigate.
Anyway, about logoText its the same: it should be enough to add it to schema. Do you mind if I do that? I have for sure another edit to apply (a different this). Moreover in this pull request you are bringing you edits to `fieldsArray.js`, which I'm not sure yet of.

Reply:
Yes. Please do. Just reject this PR and implement it the way you feel is best. You know the architecture far better than I have been able to work out for the sort time I've spent looking at it.

Reply:
We have a deal! Will do ASAP. 👍

Reply:
Whatever my issues was with the backgroundColor the other day, I cannot replicate it now. I've also modified schema.js locally as you pointed out was the best option. Works perfect. So I'll await your update for that.